### PR TITLE
[onert] Separate IOperation from the Operation header

### PR DESCRIPTION
**Pull request by ragmani**
This commit separates IOperation from the Operation header.

ONE-DCO-1.0-Signed-off-by: ragmani <ragmani0216@gmail.com>

### CODE REVIEW

**Comment by ragmani** [comment]
Waiting for merging #10875

File: runtime/onert/core/include/ir/IOperation.h [hunk 1]
```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd. All Rights Reserved
```
Comment (glistening) [inline runtime/onert/core/include/ir/IOperation.h:2]:
```suggestion
 * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
```

**Comment by ragmani** [comment]
This PR is ready for review.

**Review by glistening — APPROVED** [review]
LGTM

**Review by hseok-oh — APPROVED** [review]

**Pull request merged**